Algebraic_foundations/include/CGAL/number_utils.h

```diff
@@ -324,6 +324,36 @@ NT approximate_sqrt(const NT& nt)
   return approximate_sqrt(nt, Sqrt());
 }
 
+template <class NT>
+Comparison_result
```

Comment:
Maybe:
```suggestion
 typename Same_uncertainty_nt<Comparison_result, NT>::type
```

along with 
```
#include <CGAL/Kernel/Same_uncertainty.h>
```
at the top of the file.

Reply:
Can you just do it.